mm2src/coins/coins_tests.rs

```diff
@@ -43,6 +32,9 @@ pub fn test_import_address() {
             URL_SAFE
         ))),
         event_handlers: Default::default(),
+        request_id: 0u64.into(),
+        list_unspent_in_progress: false.into(),
+        list_unspent_subs: AsyncMutex::new(Vec::new()),
```

Comment:
Please consider to replace it with something like:
`#[cfg(test)] impl Default for NativeClientImpl {..}`
`Arc::new(NativeClientImpl { coin_ticker: "TEST".into(), ..NativeClientImpl::default() }`
Or implement Default for NativeClientImpl without `test` attribute checking